client/app.html

```diff
@@ -11,6 +11,7 @@
     <script src="../third_party/codemirror-5.19.0/addon/edit/matchbrackets.js"></script>
     <script src="../third_party/ui-codemirror-0.3.0/ui-codemirror.min.js"></script>
     <script src="app.js"></script>
+    <script src="server_url.js"></script>
```

Comment:
Perhaps the new file should be named serverUrl.js to conform with all the others (like checkBalancedParentheses.js)?